### DISCUSSION TITLE
Favicon not showing with redirect

### DISCUSSION THREAD

**Discussion by mmazzoli-dev** [vercel/vercel#6096] [Help]
I'm using vercel's method to redirect my naked domain name, to a version with www. This seems to cause my site's favicons to not work (I'm using react and it just shows the default react atom instead). I don't think it's anything in my project as the favicons all work when using one of vercel's generated domains (*.vercel.app). Guess it's something to do with the redirect? Fairly new to all this. Anyone else had this problem? Cheers.

**Comment by fu-sen** [comment]
We don't seem to have any information on this question to answer. The favicon issue doesn't seem to be Vercel immediately.
Is it possible to publish the specific domain name and favicon URL for your project so that we can investigate these specifically?

**Reply by mmazzoli-dev** [reply]
Sure, so when I use the domain marcobot.vercel.app, the favicons work fine everywhere. But when I use the domain www.marcobot.com, then some of them work, but some not (in many cases/browsers, it defaults to the react atom). The domain name is provided by google domains, and I'm using the CNAME entry for www, as specified in the vercel setup. Thank you. Apologies if this isn't vercel related... I'm new(ish) to web dev. I tried to find some info based on the redirect, but didn't manage to.
